README.md

```diff
@@ -105,6 +105,67 @@ $event = ['purchase' => ['item' => 'Golden Elephant']];
 $client->addEvent('purchases', $event);
 ```
 
+#### Data Enrichment
+A data enrichment is a powerful add-on to enrich the data you're already streaming to Keen IO by pre-processing the data and adding helpful data properties. To activate add-ons, you simply add some new properties within the "keen" namespace in your events. Detailed documentation for the configuration of our add-ons is available [here](https://keen.io/docs/api/?php#data-enrichment).
+
+Here is an example of using the [URL parser](https://keen.io/docs/streams/url-enrichment/):
+
+```php
+$client->addEvent('requests', [
+    'page_url' => 'http://my-website.com/cool/link?source=twitter&foo=bar/#title',
+    'keen' => [
+        'addons' => [
+            [
+                'name' => 'keen:url_parser',
+                'input' => [
+                    'url' => 'page_url'
+                ],
+                'output' => 'parsed_page_url'
+            ]
+        ]
+    ]
+])
```

Comment:
Could you add semicolons at the end of each?

Reply:
My bad, added! 😄